java/src/main/native/ai_onnxruntime_OnnxTensor.c

```diff
@@ -213,20 +243,23 @@
  */
 JNIEXPORT jfloat JNICALL Java_ai_onnxruntime_OnnxTensor_getFloat
   (JNIEnv * jniEnv, jobject jobj, jlong apiHandle, jlong handle, jint onnxType) {
-    (void) jobj; // Required JNI parameter not needed by functions which don't need to access their host object.
-    const OrtApi* api = (const OrtApi*) apiHandle;
-    if (onnxType == 9) {
-        uint16_t* arr;
-        checkOrtStatus(jniEnv,api,api->GetTensorMutableData((OrtValue*)handle,(void**)&arr));
-        jfloat floatVal = convertHalfToFloat(*arr);
-        return floatVal;
-    } else if (onnxType == 10) {
-        jfloat* arr;
-        checkOrtStatus(jniEnv,api,api->GetTensorMutableData((OrtValue*)handle,(void**)&arr));
-        return *arr;
-    } else {
-        return NAN;
+  (void) jobj;  // Required JNI parameter not needed by functions which don't need to access their host object.
+  const OrtApi* api = (const OrtApi*) apiHandle;
+  if (onnxType == 9) {
```

Comment:
>9) [](http://example.com/codeflow?start=18&length=2)

Can this be enum ?

Reply:
It is in Java, but these numbers are only in there. As far as I can tell the C enums don't expose the ordinal and I didn't want to have yet another copy of the enum I'd need to keep in sync.

Reply:
Fixed.

Reply:
C enums are ints with values in the order or declaration. But they can be assigned any value you wish.